### ISSUE TITLE
Default values for entity fields

### ISSUE THREAD

**Issue by yuval-hazaz** [amplication/amplication#1045]
Add option to set default values to Entity Fields

**Comment by kurdi-dev** [comment]
hoping to see this happen!

**Comment by yuval-hazaz** [comment]
@GreenMachine01 let's add this to our backlog as part of extending support for more schema flags and attributes

**Comment by yuval-hazaz** [comment]
We support this feature using custom attributes - users can use the Prisma attribute @default() and use all the different options that Prisma supports. 

We should document this in out docs "Use Default Value on Entity Fields" and explain how to do it. 

Although we can add support for a built-in text field that will be converted to an attribute, I think it will be very time-consuming and hard to maintain support for all different options, including functions like now() etc.

I suggest leaving this open for the user to use the `Custom Attributes` field instead. We can also add an example in the tooltip on using the field to configure a default value.


@GreenMachine01  please review and decide whether we should close this and pass the tasks needed for the docs to @dericksozo

**Comment by GreenMachine01** [comment]
@yuval-hazaz I agree that it sounds like we can park this one for now. Creating a task for documentation.
